plugins/@grouparoo/eloqua/README.md

```diff
@@ -0,0 +1,3 @@
+## Attribution
```

Comment:
There's a little more to add to each README now.  Check out https://github.com/grouparoo/grouparoo/blob/main/plugins/@grouparoo/mysql/README.md for an example.  Basically, we want our NPM pages to look good.